diff -Naur a/internal/lua/lua.go b/internal/lua/lua.go
--- b/internal/lua/lua.go
+++ a/internal/lua/lua.go
@@ -18,6 +18,7 @@
 	"sync"
 	"time"
 	"unicode/utf8"
+  "syscall"
 
 	humanize "github.com/dustin/go-humanize"
 	lua "github.com/yuin/gopher-lua"
@@ -44,6 +45,8 @@
 	switch pkg {
 	case "fmt":
 		return importFmt()
+	case "syscall":
+		return importSyscall()
 	case "io":
 		return importIo()
 	case "io/ioutil", "ioutil":
@@ -104,6 +107,14 @@
 
 	return pkg
 }
+
+func importSyscall() *lua.LTable {
+	pkg := L.NewTable()
+
+	L.SetField(pkg, "Exec", luar.New(L, syscall.Exec))
+
+	return pkg
+}
 
 func importIo() *lua.LTable {
 	pkg := L.NewTable()
